pkg/cloudflare/workerroute/import.go

```diff
@@ -0,0 +1,50 @@
+package workerroute
+
+import (
+	"fmt"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kubeflare/pkg/apis/crds/v1alpha1"
+	"github.com/replicatedhq/kubeflare/pkg/internal"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func FetchWorkerRoutesForZone(token string, zone string, zoneID string) ([]*v1alpha1.WorkerRoute, error) {
+	cf, err := cloudflare.NewWithAPIToken(token)
+	if err != nil {
+		return nil, errors.Wrap(err, "create clouflare client")
+	}
+
+	resources, err := cf.ListWorkerRoutes(zoneID)
+	if err != nil {
+		return nil, errors.Wrap(err, "fetch resources")
+	}
+
+	workerRoutes := []*v1alpha1.WorkerRoute{}
+	var count int
+	for _, resource := range resources.Routes {
```

Comment:
We can do `for i, resource := range resources.Routes` here and use `i` in `fmt.Sprintf("workerroute-%d", i)`.
So we won't need an extra variable